src/Lykke.Quintessence.Rootstock.DomainServices/Strategies/RootstockAddChecksumStrategy.cs

```diff
@@ -0,0 +1,56 @@
+using System.Text;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Lykke.Quintessence.Core.Blockchain;
+using Lykke.Quintessence.Core.Crypto;
+
+namespace Lykke.Quintessence.Domain.Services.Strategies
+{
+    [UsedImplicitly]
+    public class RootstockAddChecksumStrategy : IAddChecksumStrategy
+    {
+        private readonly IChainId _chainId;
+        private readonly IHashCalculator _hashCalculator;
+
+        
+        public RootstockAddChecksumStrategy(
+            IChainId chainId,
+            IHashCalculator hashCalculator)
+        {
+            _chainId = chainId;
+            _hashCalculator = hashCalculator;
+        }
+
+        
+        public async Task<string> ExecuteAsync(
+            string address)
+        {
+            address = address.Remove(0, 2).ToLowerInvariant();
```

Comment:
check address to be null?

Reply:
This strategy is never used with data, that we got from external sources.